src/idstore.js

```diff
@@ -1,18 +1,17 @@
 'use strict'
 
-const Block = require('ipld-block')
```

Comment:
there is a `Block` in multiformats, it's not quite as featureful as ipld-block IIRC or what we tried with https://github.com/ipld/js-block, but it's a really nice container `cid:bytes:value` that can be quite useful with some extra sugar that might be close to what the user previously expected. We could return that to the user and they get to consume it as they want.

See https://github.com/multiformats/js-multiformats/blob/master/src/block.js (I don't think it got documented yet because we were working on getting it _right_ and figuring out the tradeoffs between that and what we might do in js-block if we did that above it, but it should be good to use as is).